packages/forms-web-app/src/views/appeals/your-appeals.njk

```diff
@@ -73,7 +73,7 @@
                                 Continue
                               </a>
                             {% else %}
-                              <a href="#" class="govuk-link">
+                              <a href="{{item.nextDocumentDue.baseUrl}}" class="govuk-link">
```

Comment:
do we need a check for when there isn't a nextDocumentDue to avoid displaying an empty link for HAS appeals for example? Or is that handled by toDoAppeals?

Reply:
It's handled when determining whether to display a document on todo (displayDocumentOnToDo function in dashboard-functions)